src/components/CreateCourse/components/AuthorItem/AuthorItem.jsx

```diff
@@ -0,0 +1,20 @@
+import { Button } from '../../../../common';
+import {
+	ACTIONS,
+	ADD_AUTHOR_BTN_TXT,
+	DELETE_AUTHOR_BTN_TXT,
+} from '../../../../constants';
+import { Card } from 'react-bootstrap';
+
+const AuthorItem = ({ name, action, onClick }) => (
+	<div className='d-flex justify-content-between'>
+		<Card.Text>{name}</Card.Text>
+		{action === ACTIONS.ADD ? (
+			<Button text={ADD_AUTHOR_BTN_TXT} onClick={onClick} />
+		) : (
+			<Button text={DELETE_AUTHOR_BTN_TXT} onClick={onClick} />
+		)}
+	</div>
+);
```

Comment:
It would be better to remove ACTIONS dependency and pass button text as a prop to make this component more reusable.

```suggestion
const AuthorItem = ({ name, actionLabel, onClick }) => (
	<div className='d-flex justify-content-between'>
		<Card.Text>{name}</Card.Text>
		<Button text={actionLabel} onClick={onClick} />
	</div>
);
```